website/source/docs/guides/kuberenetes-deployment.html.md

```diff
@@ -0,0 +1,177 @@
+ui: 
+  service: 
+    type: "LoadBalancer" 
+```
+
+This file renames your datacenter, enables catalog sync, and sets up a load
+balancer service for the UI. The catalog sync parameters will allow you to see
+the Kubernetes services in the Consul UI. 
+
+### Initiate Rolling Upgrade 
+
+Finally, initiate the
+[upgrade](https://www.consul.io/docs/platform/k8s/run.html#upgrading-consul-on-kubernetes)
+with `helm upgrade` and the `-f` flag that passes in your new values file. This
+processes should also be quick, less than a minute.  
+
+```sh
+$ helm upgrade consul -f values.yaml 
+```
+
+You can now use `kubectl` to discover the external IP of your Consul UI.
+
+```sh 
+$ kubectl get services 
+NAME                            TYPE           CLUSTER-IP     EXTERNAL-IP             PORT(S)        AGE 
+consul                          ExternalName   <none>         consul.service.consul   <none>         11d 
+kubernetes                      ClusterIP      122.16.14.1    <none>                  443/TCP        137d
+mollified-robin-consul-dns      ClusterIP      122.16.14.25   <none>                  53/TCP,53/UDP  13d
+mollified-robin-consul-server   ClusterIP      None           <none>                  8500/TCP       13d
+mollified-robin-consul-ui       LoadBalancer   122.16.31.395  36.276.67.195           80:32718/TCP   13d
+```
+
+The service should have `consul-ui` appended to the deployment name. Note, you
+do not need to specify a port when accessing the dashboard. 
+
+## Access Consul 
+
+In addition to accessing Consul with the UI, you can manage Consul with the
+HTTP API or by directly connecting to the pod with `kubectl`. 
+
+### Kubectl
+
+To access the pod and data directory you can start a shell with `kubectl`.
+
+```sh 
+$ kubectl exec -it mollified-robin-consul-server-0 /bin/ash 
+```
+
+This will allow you to navigate the file system and run Consul CLI commands on
+the pod. For example you can view the Consul members. 
+
+```sh 
+$ consul members 
+Node                                   Address           Status  Type    Build  Protocol  DC        Segment 
+mollified-robin-consul-server-0        172.20.2.18:8301  alive   server  1.4.2  2         hashidc1  <all>
+mollified-robin-consul-server-1        172.20.0.21:8301  alive   server  1.4.2  2         hashidc1  <all> 
+mollified-robin-consul-server-2        172.20.1.18:8301  alive   server  1.4.2  2         hashidc1  <all>
+gke-tier-2-cluster-default-pool-leri5  172.20.1.17:8301  alive   client  1.4.2  2         hashidc1  <default>
+gke-tier-2-cluster-default-pool-gnv4   172.20.2.17:8301  alive   client  1.4.2  2         hashidc1  <default>
+gke-tier-2-cluster-default-pool-zrr0   172.20.0.20:8301  alive   client  1.4.2  2         hashidc1  <default>
+```
+
+### Consul HTTP API
+
+You can use the Consul HTTP API by communicating to the local agent running on
+the Kubernetes node. The process requires the installation and configuration of
+an additional kubernetes API. You can read the
```

Comment:
```suggestion
an additional Kubernetes API. You can read the
```

Also, I don't think this is referring to another API that needs installation. It makes use of the k8s [downward api](https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/), but that is available to any Kubernetes resource you create (from the linked Consul docs: "In addition to pods, anything with a pod template can also access the downward API and can therefore also access Consul: StatefulSets, Deployments, Jobs, etc."). Do you mean that the user needs to set up an additional deployed Kubernetes object to use the HTTP API?

Reply:
I think I just misread that section of the docs. Let me update the wording.